components/welcome-mat/info-badge.jsx

```diff
@@ -0,0 +1,103 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// Implements the [Welcome Mat InfoBadge design pattern](https://lightningdesignsystem.com/components/welcome-mat/) in React.
+// Based on SLDS v2.4.0
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from '~/components/icon';
+
+// ### shortid
+// [npmjs.com/package/shortid](https://www.npmjs.com/package/shortid)
+// shortid is a short, non-sequential, url-friendly, unique id generator
+import shortid from 'shortid';
+import { WELCOME_MAT_TILE } from '../../utilities/constants';
+
+const displayName = WELCOME_MAT_TILE;
```

Comment:
This needs to be `WELCOME_MAT_INFO_BADGE`, imported from constants above

Reply:
Yeah, that seems like I made a typo :(